tests/Fixtures/Entity/FooToStringNull.php

```diff
@@ -16,7 +16,7 @@
 final class FooToStringNull
 {
     // In case __toString returns an attribute not yet set
-    public function __toString()
+    public function __toString(): ?string
```

Comment:
A toString method cant return null 🤔

Reply:
We have a tostring method returning null because it was possible in php7 versions. I didn't know how to solve the psalm issue, I'll suppress it instead.